Modules/Core/Common/test/itkBSplineInterpolationWeightFunctionTest.cxx

```diff
@@ -145,6 +145,16 @@ itkBSplineInterpolationWeightFunctionTest(int, char *[])
 
     auto function = FunctionType::New();
 
+#if !defined(ITK_LEGACY_REMOVE)
```

Comment:
I guess this sneaked in from the main PR. Either remove or put in a separate commit 😄

Reply:
Oops. Compile error says it needs to be removed.